admin/metabox/class-metabox.php

```diff
@@ -341,6 +337,23 @@ private function get_seo_meta_section() {
 			$label = '<span class="wpseo-score-icon-container" id="wpseo-seo-score-icon"></span>' . $label;
 		}
 
+		$html_after = '';
+
+		if ( WPSEO_Capability_Utils::current_user_can( 'wpseo_edit_advanced_metadata' ) || WPSEO_Options::get( 'disableadvanced_meta' ) === false ) {
+			$advanced_collapsible = new WPSEO_Paper_Presenter(
+				__( 'Advanced', 'wordpress-seo' ),
+				null,
+				array(
+					'collapsible' => true,
+					'class'       => 'metabox wpseo-form wpseo-collapsible-container',
+					'content'     => $this->get_tab_content( 'advanced' ),
+					'paper_id'    => 'collapsible-advanced-settings',
+				)
+			);
+
+			$html_after = '<div class="wpseo_content_wrapper">' . $advanced_collapsible->get_output() . '</div>';
```

Comment:
The other collapsibles (SEO and readability) have their ID on the button, while this PR puts the advanced collapsible ID on the div surrounding the h2 and button. Let's go for consistency (also for the social collapsibles.